docs/custom-rules.md

```diff
@@ -113,6 +113,7 @@ class RulesConfig:
     bbr_rules_dirs: Optional[List[Path]] = field(default_factory=list)
     bypass_version_lock: bool = False
     exception_dir: Optional[Path] = None
+    normalize_kql_keywords: bool = True
```

Comment:
I saw a comment saying we have to do a docs update. Will that also highlight what this field does in this file?

Reply:
Yes, the docs update will highlight this :+1:

Reply:
Looked through the existing docs, I did not see a place where we talk about the different config options available. I think it would be worth adding along with the documenting the other options as well. Made an issue to track: https://github.com/elastic/DaC-Reference/issues/15. Decoupled from this PR.